src/fixtures/signup.fixtures.js

```diff
@@ -0,0 +1,11 @@
+import faker from 'faker';
```

Comment:
I gave feedback that the format for the fixture should be `<model name>.fixture.js`. E.g. `user.fixture.js`. And you should have a named export since multiple test data will be inside this file.

Reply:
The file name should be prefixed with `.fixture.js` and not `.fixtures.js`.